unity-renderer/Assets/DCLPlugins/BuilderInWorld/Publisher/ProjectPublishHUD/Scripts/Projects/PublishProjectProgressView.cs

```diff
@@ -136,14 +140,17 @@ private IEnumerator FakePublishProgress()
         {
             while (true)
```

Comment:
How does this while (true) loop execution stop? is this `FakePublishProgress()` being used?